docs/modules/utils/combine_docs_examples/vectorstores.ipynb

```diff
@@ -719,13 +719,244 @@
     "docs[0]"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## OpenSearch\n",
+    "To run, you should have the opensearch instance up and running: [here](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/)\n",
+    "`similarity_search` by default performs the Approximate k-NN Search which uses one of the several algorithms like lucene, nmslib, faiss recommended for\n",
+    "large datasets. To perform brute force search we have other search methods known as Script Scoring and Painless Scripting.\n",
+    "Check [this](https://opensearch.org/docs/latest/search-plugins/knn/index/) for more details.\n"
```

Comment:
Can we skip custom scoring for now?

Reply:
Can you please tell the reason for skipping it?

Reply:
Elasticsearch is supporting the [script scoring](https://github.com/hwchase17/langchain/blob/master/langchain/vectorstores/elastic_vector_search.py#L22-L31). Of course it is the only and default search they are using here. So, IMO it is good to include.